code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm

```diff
@@ -68,10 +68,12 @@
 
 	succeed_activate()
 	add_cooldown()
-
-/mob/living/carbon/xenomorph/proc/salve_healing()	//Slight modification of the heal_wounds proc
+	
+//Slight modification of the heal_wounds proc
+/mob/living/carbon/xenomorph/proc/salve_healing()	

```

Comment:
```suggestion

/mob/living/carbon/xenomorph/proc/salve_healing() //Slight modification of the heal_wounds proc
```

Reply:
this was fine as it was

Reply:
If you intended for this to be docs, you could clean up the statement to be clearer.
You also need to use 3 `///`.